docs/page-software-operation-view.md

```diff
@@ -1,3 +1,430 @@
+              "odrl:operator": {
+                "@id": "odrl:eq"
+              },
+              "odrl:rightOperand": "active"
+            },
+            {
+              "@type": "Constraint",
+              "odrl:leftOperand": "FrameworkAgreement.sustainability",
+              "odrl:operator": {
+                "@id": "odrl:eq"
+              },
+              "odrl:rightOperand": "active"
+            }
+          ]
+        }
+      }
+    ]
+  }
+}
+
+```
+
+
+### Contract Definition Configuration
+
+Contract definitions allow us to expose the assets and link them to a contract policy and a access policy.
+
+> **_INFO:_** *Remember that all **policies and assets** you bind to a contract **must be defined in the same EDC Connector** and linked though their ID in the configuration from the contract.*
+
+#### Variables:
+
+| Name | Description | Example Value                                                                                                                                 |
+| ---- | -------- |-----------------------------------------------------------------------------------------------------------------------------------------------|
+| ContractDefinitionId | UUID that identifies the policy in the EDC Connector | 76b50bfc-ec19-457f-9911-a283f0d6d0df                                                                                                          |
+| AssetId | Combination of Digital Twin and Sub Model UUIDs | **Example value for asset**: 32aa72de-297a-4405-9148-13e12744028a-699f1245-f57e-4d6b-acdb-ab763665554a <br/> **Example value for registry**: digital-twin-registry |
+| AccessPolicyId | Policy that allows/restricts/enforces asset access constrains | ad8d2c57-cf32-409c-96a8-be59675b6ae5                                                                                                          |
+| ContractPolicyId | Policy that allows/restricts/enforces contract constrains | ad8d2c57-cf32-409c-96a8-be59675b6ae5                                                                                                          |
+
+
+#### Format and Fields:
+
+> **_INFO:_** *For testing proposes and in order to ease the access to your assets we are going to define the **same policy as accessPolicy and as contractPolicy**. However, you are recommended to configure two separated policies and specify them adapting each one of them to your specific needs.*
+
+```json
+{
+    "@context": {},
+    "@id": "{{ContractDefinitionId}}",
+    "@type": "ContractDefinition",
+    "accessPolicyId": "{{AccessPolicyId}}",
+    "contractPolicyId": "{{ContractPolicyId}}",
+    "assetsSelector" : {
+        "@type" : "CriterionDto",
+        "operandLeft": "https://w3id.org/edc/v0.0.1/ns/id",
+        "operator": "=",
+        "operandRight": "{{AssetId}}"
+    }
+}
+```
+
+### Digital Twin Registration
+
+Once you finish the configuration, to make the endpoint public configure in the following way your Digital Twin:
```

Comment:
"to make the endpoint public configure your digital twin in the following way" ?